pkg/jobs/jobs_test.go

```diff
@@ -127,7 +127,8 @@ func (expected *expectation) verify(id jobspb.JobID, expectedStatus jobs.Status)
 	if started.Equal(timeutil.UnixEpoch) && status == jobs.StatusSucceeded {
 		return errors.Errorf("started time is empty but job claims to be successful")
 	}
-	if status == jobs.StatusRunning || status == jobs.StatusPauseRequested {
+	if status == jobs.StatusRunning ||
```

Comment:
this was under the wrap limit before, wasn't it?

Reply:
Will unwap.